srcs/parser/groups_init.c

```diff
@@ -6,31 +6,33 @@
 /*   By: dkolida <dkolida@student.42warsaw.pl>      +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/07/21 22:22:09 by dkolida           #+#    #+#             */
-/*   Updated: 2024/08/07 17:04:35 by dkolida          ###   ########.fr       */
+/*   Updated: 2024/08/07 17:11:36 by dkolida          ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
 t_group	*group_init(int argc);
-//static void		extract_infile(t_group *group);
 void	add_to_group(t_shell *shell, char *token, int is_end);
 int		shell_groups_init(t_shell *shell, char **tokens);
-void	set_group_infile_name(t_group *group, char *token);
 
 void	group_input(t_shell *shell, char **tokens)
 {
-	int		tokens_c;
-	int		i;
+	int	tokens_c;
+	int	i;
+	int	*group_i;
 
+	group_i = &shell->group_i;
```

Comment:
Is this variable necessary? I guess it increases clarity but should work the same if we simply used &shell->group_i?